### PR TITLE
chore: Improve the README of the project

### LINKED ISSUE #419: Improve the README of the project
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request

#### Feature Description

The README of the project needs a lot of improvements. Here are a few of them:

- [ ] Add the title of the project as a header at the top.
- [ ] Add badge for build status from travis.
- [ ] Divide data into relevant groups such as build instructions, branch policy etc.
- [ ] Improve the header for Contact and Legal/Disclaimer
- [ ] Add license for the project and a link to it.   
- [ ] Commit style
- [ ] Code style
- [ ] Libraries used

Please suggest something if i have missed it.

### PR DESCRIPTION
**Pull request by srv-twry**
Fixes #419 

Changes: Improved the README of the project by adding various informations.

Screenshots/GIF for the change: NA
To check the final rendered README, please switch to ```srv-twry/improvereadme``` branch.
@mhutti1 Can you suggest what to do with the ```Legal and Disclaimer``` section. It looks weird. I couldn't find the "COPYING" file.

### CODE REVIEW

**Comment by mhutti1** [comment]
@kelson42 What are your thoughts on the COPYING file?

**Comment by kelson42** [comment]
@mhutti1 It looks like the COPYING file has not been copied properly as we have moved from the old 'kiwix-xulrunner" git repo. I just have added it directly to `master`. That said each source file should have in addition the GPL3 header (like here for example https://github.com/kiwix/kiwix-xulrunner/blob/master/src/server/kiwix-serve.cpp).

**Comment by srv-twry** [comment]
@mhutti1 I think this one can be merged now. 
The only thing remaining is the Roadmap which can be added later since it will take time. 
Right now i think we should merge it because we need the contribution and commit guidelines because most issues and PR's aren't following it.

File: docs/commitstyle.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+body
+
+***
+
+### Title
+The title consists of the subject and type of the commit message.
+
+### Type
+The type is contained within the title and can be one of the following types:
```
Comment (mhutti1) [inline docs/commitstyle.md]: What are the benefits of these that aren't included in the issue labels?

Comment (srv-twry) [reply docs/commitstyle.md]: The issue labels are abstract. These one's explicitly show, what has been done in the commit. If you want i can also add these to the issue labels.

Comment (mhutti1) [reply docs/commitstyle.md]: In the past I have normally just used an imperative verb such as Fix, Improve, Implement, Remove. I am not sure what limiting our variety of these verbs adds.

Comment (srv-twry) [reply docs/commitstyle.md]: Okay. I think any thing that makes the intention of the commit clear works. What do you want me to add in here ?

Comment (mhutti1) [reply docs/commitstyle.md]: I would maybe put something like.

"Imperative verb (not past tense) such as Fix, Implement, Remove, Duplicate... describing the main actions performed by this commit"

**Comment by srv-twry** [comment]
@brijeshshah13 Yes, i prefer them too and started initially with them only, but in the examples [here](https://github.com/kiwix/kiwix-android/pull/420#discussion_r169824487) it's capitalised.

**Comment by brijeshshah13** [comment]
@srv-twry Maybe you should consult with @mhutti1 about this again because he didn't emphasize on using the first letter capital.

File: docs/codestyle.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Code Style Guidelines for Kiwix-Android
+
+Our code style guidelines is based on the [Android Code Style Guidelines for Contributors](https://source.android.com/source/code-style.html).
```
Comment (mhutti1) [inline docs/codestyle.md]: We use the Google Java Style guide not the Android style guide.

**Comment by mhutti1** [comment]
In regards to capitalisation the first letter of the commit message should be capitalised along with any others words as dictated by English.

**Comment by srv-twry** [comment]
@mhutti1 Ready for review when you get time.

**Pull request merged**